lib/suggestion-list-element.js

```diff
@@ -449,7 +449,7 @@ module.exports = class SuggestionListElement {
     }
   }
 
-  getDisplay (text, snippet, displayText, replacementPrefix) {
+  getDisplayFrag (text, snippet, displayText, replacementPrefix) {
```

Comment:
Sorry to be a pain, but we prefer to use whole words in all Atom code unless there's a very special case. (Like "id")

Reply:
🙈